{"name":"Local: exception_handliing1","url":"c:\\codes\\exception_handliing1.cpp","tests":[{"id":1667404159523,"input":"8\n4","output":""}],"interactive":false,"memoryLimit":1024,"timeLimit":3000,"srcPath":"c:\\codes\\exception_handliing1.cpp","group":"local","local":true}